codegen/templates/tchannel_endpoint.tmpl

```diff
@@ -109,15 +126,24 @@ func (h *{{$handlerName}}) Handle(
 			{{$method := .Name -}}
 			{{range .Exceptions -}}
 				case *{{.Type}}:
+					h.endpoint.Logger.Error(
```

Comment:
this log needs to be inside the if statement.

Reply:
This should be under the `case` since it logs the appropriate `.Type` error message.